### DISCUSSION TITLE
Compter le nombre de caractères dans un champ texte au fur et mesure que l'utilisateur saisie.

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#295] [Annonces]
Bonjour,

Est-ce qu'il existe une fonctionnalité d'où au fur et mesure l'utilisateur saisie du texte, un compteur du nombre est affiché ?

N.B. : J'ai regardé la documentation et je n'ai rien trouvé.

**Comment by Hutson48** [comment]
Bonjour, 

Actuellement, il n'existe pas de fonctionnalité permettant d'afficher un compteur du nombre de caractères ou de mots au fur et à mesure de la saisie de texte. Cependant, c'est une très bonne idée ! Nous allons la prendre en considération et vous tiendrons informé dès qu'une solution sera développée en ce sens.

Merci pour votre retour.
